numpy/lib/tests/test_histograms.py

```diff
@@ -289,13 +289,13 @@ def test_unsigned_monotonicity_check(self):
     def test_object_array_of_0d(self):
         # gh-7864
         assert_raises(ValueError,
-            histogram, [np.array([0.4]) for i in range(10)] + [-np.inf])
+            histogram, [np.array(0.4) for i in range(10)] + [-np.inf])
         assert_raises(ValueError,
-            histogram, [np.array([0.4]) for i in range(10)] + [np.inf])
+            histogram, [np.array(0.4) for i in range(10)] + [np.inf])
 
         # these should not crash
-        np.histogram([np.array([0.5]) for i in range(10)] + [.500000000000001])
-        np.histogram([np.array([0.5]) for i in range(10)] + [.5])
+        np.histogram([np.array(0.5) for i in range(10)] + [.500000000000001])
```

Comment:
I think I wrote these mostly to test that they didn't blow up in a non-sensical internal crash. The thinking was that this is just doing a histogram on object arrays.

If we could commit to making `bool(np.array([1]) < 0)` an error, then this would no longer be allowed either.

Reply:
I see. But just to be sure: is the change here fine? If we want object arrays, it might be better to just create them explicitly.